lib/api/controllers/security.js

```diff
@@ -305,6 +305,7 @@ class SecurityController extends NativeController {
       description,
       { apiKeyId, creatorId, refresh });
 
+    this.kuzzle.log.info(`[SECURITY] User "${request.context.user._id}" applied action "${request.input.action}" on user "${userId}."`);
```

Comment:
:warning: `request.context.user` can be null, for instance if the request has been instantiated by a plugin (either with the SDK or with the execute method).

After giving it some thoughts, my guess is that you should create a dedicated `getUserId` getter in the `BaseController` class. 
This getter should return  `<null>` or something like that (to be debated) if the user object is null.
It should return neither `-1` nor `anonymous` because we don't know if the request is not a consequence of some other API call made by an authenticated user.

That new getter should be used everywhere.